Engine/AssemblyFinder.cs

```diff
@@ -97,6 +97,11 @@ private void SyncFiles()
                             {
                                 if (StrEq(subDir.Name, "obj"))
                                     continue; // skip obj subfolder
+                                if (subDir.GetFiles("OpenTap.dll").Any())
+                                {
+                                    log.Warning("Found OpenTap installation in subdirectory: '{0}'", subDir.Name);
```

Comment:
We need to inform the user that plugins will not be loaded from this subdirectory since this is new behavior. Nested installations can cause unexpected behavior and are not supported.